src/pages/programs/styles/MarkAttendance.css

```diff
@@ -0,0 +1,90 @@
+:root {
+    --text-link: #888b8f;
+    --button-color: #e8e9eb;
+    --mid-color: #d0d0d1;
+}
```

Comment:
careful defining these variables - i would put this in globals.css since they will be defined everywhere